pallets/ajuna-awesome-avatars/src/lib.rs

```diff
@@ -40,15 +41,60 @@ pub mod pallet {
 		type Event: From<Event<Self>> + IsType<<Self as frame_system::Config>::Event>;
 	}
 
-	// Add pallet storage here
-
 	#[pallet::event]
-	// #[pallet::generate_deposit(pub(super) fn deposit_event)]
-	pub enum Event<T: Config> {}
+	#[pallet::generate_deposit(pub(super) fn deposit_event)]
+	pub enum Event<T: Config> {
+		/// A new organizer has been set
+		OrganizerSet { organizer: T::AccountId },
```

Comment:
I think OrganizerSet is a better name for the event since we are talking about something that occurred in the immediate past, so I'll leave the name like this for now.

Reply:
You're right. I just realised other events are also named this way.